app/controllers/api/v1/private_items_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PrivateItemsController < ApplicationController
```

Comment:
Instead of creating a `PrivateItemsController` for testing/mockup, i think we also could define a private controller that potentially will be used in the next tickets, so we don't need to removed it after this PR.

Reply:
I think the method will be temporary, even with a feature-ready controller, I find it easier to keep the unused soon-to-be-removed code in the same place for easier clean up 🙏

Reply:
that's fine.